broker/README.md

```diff
@@ -2,106 +2,26 @@
 
 ![RabbitMQ Broker for Knative Eventing](rabbitmq-knative-broker.png)
 
-## Prerequisites
 
-install Knative Serving and Eventing as documented
-[here](https://knative.dev/docs/install/any-kubernetes-cluster/)
-
-If you have an existing RabbitMQ cluster that you'd like to use, you can skip
-installing the RabbitMQ Cluster operator.
-
-If you do not have a RabbitMQ cluster, you can install the RabbitMQ Operator as
-documented [here](https://github.com/rabbitmq/cluster-operator) which makes it
-easy to spin up a RabbitMQ cluster. At least version v1.0.0 is required.
 
 ## Installation
 
-You can install a released version of
-[Knative RabbitMQ Broker](https://github.com/knative-sandbox/eventing-rabbitmq/releases/)
+We provide two versions of the RabbitMQ Knative Eventing Broker.
 
-For example, if you wanted to install version v0.19.0 you would run:
+1. Standalone Broker
+2. RabbitMQ operator based Broker
 
-```shell
-kubectl apply --filename https://github.com/knative-sandbox/eventing-rabbitmq/releases/download/v0.19.0/rabbitmq-broker.yaml
-```
+### Standalone Broker
 
-If you want to run the latest version from this repo, you can use
-[`ko`](https://github.com/google/ko) to install it.
+This Broker works by utilizing libraries to manage RabbitMQ resources directly and as name implies does not have have dependencies on other operators. Choose this if you do not manage your clusters lifecycle with the [Cluster Operator](https://github.com/rabbitmq/cluster-operator).
 
-```
-ko apply -f config/broker/
-```
+[Install Standalone Broker](./standalone.md)
 
-## Creating a Knative RabbitMQ Broker
+### RabbitMQ operator based Broker
```

Comment:
Would it be useful to suggest this in case the person looking at the instructions doesn't know enough or have a preference ? The instructions for this are longer so my noob self would likely try to figure out the standalone setup 😁